src/hammer-vlsi/defaults.yml

```diff
@@ -608,3 +619,51 @@ technology.core:
     # Can be a wildcard/glob
     # type: str
     tap_cell_rail_reference: null
+
+sim.inputs:
+  # Top RTL module
+  top_module: null
+  # top_module_config: null
+  input_files: []
+  # Debug option
+  debug: false
+  # Black box command line option escape hatch
+  options: [] 
+  # Define options
+  defines: []
+  # C compiler options
+  compiler_opts: []
+  # Simulation timescale
+  timescale: null
+  # C source files 
+  csrc: []
```

Comment:
this is usually simulator-specific, so IMO it should go in the tool plugin namespace